### ISSUE TITLE
Disengaging during combat.

### ISSUE THREAD

**Issue by Jinko1** [EasyFarm/EasyFarm#405]
Since the last update the bot drops a target (disengages) during combat and then picks it back up, anyone else experiencing this ?

**Comment by adeptio** [comment]
#400 is specific to playing with other player characters in the same party.  I believe the disengage, re-engage is part of the fix for #358.

**Comment by Jinko1** [comment]
Fair enough, however this can cause death either to yourself or your trust as FF11 is incredible slow at disengaging and re-engaging, especially if you are fighting something fairly tough.
It also allows others to steal your claimed targets.

**Comment by Mykezero** [comment]
Might need to fine tune the disengage / re-engage behavior so that it doesn't happen when receiving the first "Unable to see" message. Maybe it's too sensitive?

**Comment by Jinko1** [comment]
I think that would definitely help.
